whiteboard/templates/whiteboard.pug

```diff
@@ -1,9 +1,9 @@
-extends base.pug
+extends ../base.pug
 block style
-  link(rel='stylesheet' type='text/css' href='css/whiteboard.css')
+  link(rel='stylesheet' type='text/css' href='whiteboard/css/whiteboard.css')
 
```

Comment:
These path changes are because templates and static now have whiteboard/ subfolders (links), so we have to go static/whiteboard/css/... and static/whiteboard/js... not just static/css... And base.pug is in templates/ while whiteboard.pug is in templates/whiteboard/ so for whiteboard.pug to find base.pug, we need to do ../base.pug so it looks in the parent directory 😕